lib/jelix/plugins/formwidget/date_html/date_html.formwidget.php

```diff
@@ -40,13 +40,13 @@ protected function outputJs() {
         if($maxDate)
             $js .= "c.maxDate = '".$maxDate->toString(jDateTime::DB_DFORMAT)."';\n";
 
+        $this->parentWidget->addJs($js);
+        $this->commonJs();
+
         if($ctrl instanceof jFormsControlDate || get_class($ctrl->datatype) == 'jDatatypeDate' || get_class($ctrl->datatype) == 'jDatatypeLocaleDate'){
             $config = isset($ctrl->datepickerConfig)?$ctrl->datepickerConfig:jApp::config()->forms['datepicker'];
             $js .= 'jelix_datepicker_'.$config."(c, jFormsJQ.config);\n";
```

Comment:
since $this->parentWidget->addJs() is already called because of the move of the lines of code, you cannot add something in $js anymore. You should do a $this->parentWidget->addJs() again here